### ISSUE TITLE
Car keypoint epoch loss standstill

### ISSUE THREAD

**Issue by marco-lee25** [openpifpaf/openpifpaf#591]
Hi, I am using openpifpaf==0.13.7 for training the 24-car keypoint detection. As the pre-trained "shufflenetv2k16-apollo-24", use the CompositeField3 while the 0.13.x version has updated the structure to CompositeField4, and I have to train a new model. While I using the apollo3D data for training, the loss is not decreasing after some epochs.

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/72645115/215008544-76b831c1-0d02-47d3-a8e7-b423bb70bb8f.png">

 I followed the car key points tutorial on the https://openpifpaf.github.io/plugins_apollocar3d.html#train and made use of the command :
python3 -m openpifpaf.train --dataset apollo \
--basenet=shufflenetv2k16 --apollo-square-edge=769 \
--lr=0.00002 --momentum=0.95  --b-scale=5.0 \
--epochs=300 --lr-decay 160 260 --lr-decay-epochs=10  --weight-decay=1e-5 \
--weight-decay=1e-5  --val-interval 10 --loader-workers 16 --apollo-upsample 2 \
--apollo-bmin 2 --batch-size 8

Could you give me some advice on this issue? I also tried different learning rates, momentum etc. But it turns out that the epoch loss is still very high.

**Comment by shteren1** [comment]
Hi @marco-lee25 do you happen to still have the training data? it seems like the download links for apolloscape dataset are broken for over a year and there is no word from the maintainers.

**Comment by xu-jiusi** [comment]
我有训练数据但是好像不全，缺少一个objloader.py
